Source/GSStoryboardTransform.m

```diff
@@ -0,0 +1,828 @@
+                                                                               sender: src
+                                                                                 kind: kind];
+              
+              NSUInteger count = [[objects children] count];
+              [objects insertChild: sbproxy
+                           atIndex: count - 1];
+              
+              // add action to parent ONLY if it is NOT a controller..
+              if (![segue_parent_parent_name isEqualToString: @"windowController"] &&
+                  ![segue_parent_parent_name isEqualToString: @"viewController"])
+                {              
+                  // Create action...
+                  NSXMLElement *action = [NSXMLElement elementWithName: @"action"];
+                  NSXMLNode *selector
+                    = [NSXMLNode attributeWithName: @"selector"
+                                       stringValue: @"doAction:"];
+                  NSXMLNode *target
+                    = [NSXMLNode attributeWithName: @"target"
+                                       stringValue: [[sbproxy attributeForName: @"id"] stringValue]];
+                  NSXMLNode *controller_ident
+                    = [NSXMLNode attributeWithName: @"id"
+                                       stringValue: uid]; 
+                  [action addAttribute: selector];
+                  [action addAttribute: target];
+                  [action addAttribute: controller_ident];
+                  [segue_parent addChild: action];
+                }
+              
+              // Create the segue...
+              NSStoryboardSegue *ss = [[NSStoryboardSegue alloc] initWithIdentifier: ident
+                                                                             source: src
+                                                                        destination: dst];
+              [ss _setKind: kind];
+              [ss _setRelationship: rel];
+              
+              // Add to maptable...
+              [mapTable setObject: ss
+                           forKey: ident];
+              
+            } // only process segue objects...
+        } // iterate over objects in each set of connections
+      END_FOR_IN(children);          
+    } // iterate over connection objs
+  END_FOR_IN(connectionsArray);
+
+  return mapTable;
+}
+
+- (void) processSegues: (NSXMLDocument *)xml
+       forControllerId: (NSString *)identifier
+{
+  BOOL processed = [self isProcessedDocument: xml];
+  if (processed)
+    {
+      return; // don't processed the document again
+    }
+  else // Get the controller... there is only one per scene.
+    {
+      NSArray *array = [xml nodesForXPath: @"//objects[1]"
+                                    error: NULL];
+      NSXMLElement *objects = [array objectAtIndex: 0]; // get the "objects" section
+      NSString *src = [self controllerIdWithDocument: xml];                            
```

Comment:
Looking at the code in processStoryboard: I would say that the value of "identifier" is always the same as the one you get here for "src" except for the application case. If that difference is important it could be resolved easily here.

Reply:
Eliminated the call.  It is, indeed, unnecessary.